apps/common/docker-compose.tests.yml

```diff
@@ -13,6 +13,12 @@ services:
             MC_DOWNLOADS_AMAZON_S3_DIRECTORY_NAME: "${MC_DOWNLOADS_AMAZON_S3_DIRECTORY_NAME}"
             MC_PUBLIC_STORE_TYPE: "postgresql"
             MC_PUBLIC_STORE_SALT: "foo"
+            # Email address to point to in List-Unsubscribe email header.
+            # Technically we don't have a straightforward "unsubscribe" endpoint, but our 
+            # emails are more likely to be marked spam if we don't have such a header, so
+            # we make the email subject "Delete account and unsubscribe" in 
+            # mediawords/util/config/common.py
+            MC_UNSUBSCRIBE_MAILTO_LINK: "support@example.com"
```

Comment:
Could you rename `MC_UNSUBSCRIBE_MAILTO_LINK` to something like `MC_EMAIL_FROM_ADDRESS` or `MC_MAIL_FROM_ADDRESS`?

* The rest of the email-related settings are prefixed with either `MC_EMAIL_` or `MC_MAIL_`
    * It's not great that we ended up with both `MC_EMAIL_` and `MC_MAIL_` env. var. prefixes, I think it was me who messed it up, but I guess fixing that is out of scope
* The env. variable getter is called `unsubscribe_address()`
* `_MAILTO_LINK` would kind of imply that instead of just the address (`support@example.com`) the variable is supposed to have the whole `mailto:` link in it (`mailto:support@example.com`), and the env. variable getter wouldn't test for that, so one could accidentally end up with a final header resulting into something like `List-Unsubscribe: mailto:mailto:support@example.com` which, depending on how attentive the receiving server is, could lead to emails being rejected due to an invalid header